Makefile

```diff
@@ -64,7 +71,7 @@ clean :
 	    $(DATADIR)/labelled.csv.gz  $(DATADIR)/filtered.csv.gz  $(DATADIR)/old_taxons.csv.gz  \
 	    $(DATADIR)/labelled_level1.csv.gz  $(DATADIR)/labelled_level2.csv.gz  \
 	    $(DATADIR)/empty_taxons_not_world.csv.gz  $(DATADIR)/new_content.csv.gz \
-	    data/taxons.json data/content.json
+	    data/taxons.json data/content.json.gz data/export_untagged_content.json.gz data/export_untagged_content.json.gz
```

Comment:
`data/export_untagged_content.json.gz` looks to be here twice, does it need to be?